src/resolve/resolved-resolv-conf.c

```diff
@@ -323,6 +323,25 @@ static int write_stub_resolv_conf_contents(FILE *f, OrderedSet *dns, OrderedSet
         return fflush_and_check(f);
 }
 
+void manager_symlink_stub_to_uplink_resolv_conf(void) {
+        _cleanup_free_ char *fname = NULL;
+        int r;
+        if (access(PRIVATE_UPLINK_RESOLV_CONF, R_OK) != 0) {
```

Comment:
i'd use F_OK here...

also we generally check for syscalls failing with a `< 0` check